class.jetpack-jitm.php

```diff
@@ -43,7 +43,7 @@ function photon_msg() {
 					</svg>
 				</div>
 				<p>
-					<?php _e( 'Deliver super-fast images to your visitors that are automatically optimized for any device.', 'jetpack' ); ?>
+					<?php _e( 'Speed up your media library and save bandwidth costs by using a free content delivery network.', 'jetpack' ); ?>
```

Comment:
I was trying to go for a message that's within the context of what the user is doing working under the assumption that in most cases stuff in the media library is to be used in posts and pages.

cc @jessefriedman